core/workspace_svg.js

```diff
@@ -1482,7 +1482,9 @@ Blockly.WorkspaceSvg.prototype.updateToolbox = function(tree) {
     this.options.languageTree = tree;
     this.flyout_.show(tree.childNodes);
   }
-  this.toolbox_.position();
```

Comment:
Hm yeah, #793 made it so that the entire toolbox DOM is recreated when populating it, which isn't true for the flyout.  So we shouldn't need to call position.

Alrighty, I'm happy with this PR.  Thanks!